src/components/Hamburger/Hamburger.js

```diff
@@ -3,25 +3,34 @@ import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
 import { hideVisually } from 'polished';
 
-const LAYER_HEIGHT = '2px';
+const LAYER_HEIGHT = '1px';
+const HAMBURGUER_WIDTH = '12px';
 
 const buttonBaseStyles = ({ theme }) => css`
   label: hamburger-button;
-  box-sizing: content-box;
+  outline: none;
```

Comment:
Do we really want to the disable the outline? How would we show the focus on keyboard navigation?